### DISCUSSION TITLE
Welcome to google-api-python-client Discussions!

### DISCUSSION THREAD

**Discussion by parthea** [googleapis/google-api-python-client#1626] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by mnstine** [comment]
Hello,

I like the idea of this discussion forum.  I have an issue with some code i am working on and may need some help in this area.  

Will post back here later when I get back to my development system.

**Comment by shreyasanand** [comment]
Is there documentation or samples on how to build and use client for secret manager?

**Reply by danoscarmike** [reply]
Hello @shreyasanand! I'd encourage you to take a look at the client at https://github.com/googleapis/python-secret-manager. You'll also find a sub-directory of sample code there that may be helpful.

**Comment by Donald0101** [comment]
Okay

**Comment by Donald0101** [comment]
Thank you very much.. hope you will guide me more on the process.
